src/core/util/stringconversion.cpp

```diff
@@ -43,27 +44,57 @@
 #include <windows.h>
 #endif
 
+#include <fmt/format.h>
+
 namespace inviwo {
 
 namespace util {
 
-#if defined(_WIN32)
 std::wstring toWstring(const std::string& str) {
+#if defined(_WIN32)
+    if (str.empty()) return std::wstring();
     int size_needed = MultiByteToWideChar(CP_UTF8, 0, str.data(), (int)str.size(), NULL, 0);
     std::wstring result(size_needed, 0);
     MultiByteToWideChar(CP_UTF8, 0, str.data(), (int)str.size(), &result[0], size_needed);
     return result;
-}
 #else
-std::wstring toWstring(const std::string& str) {
     auto state = std::mbstate_t();
-    const char* s = str.c_str();
-    size_t len = std::mbsrtowcs(nullptr, &s, 0, &state) + 1;
+    size_t len = std::mbsrtowcs(nullptr, &str.data(), 0, &state) + 1;
     std::wstring result(len, 0);
-    std::mbsrtowcs(&result[0], &s, result.size(), &state);
+    std::mbsrtowcs(result.data(), &str.data(), result.size(), &state);
```

Comment:
`mbsrtowcs` takes `const char **` so you need to pass in `&&str.data()` here and the line above. This is also true for `wcsrtombs`  
https://en.cppreference.com/w/cpp/string/multibyte/mbsrtowcs
https://en.cppreference.com/w/cpp/string/multibyte/wcsrtombs

Reply:
that's not the issue, problem is str.data() returns a const char* r-value which I can not take the adress of.. need to assign it to an l-value first...